packages/components-js/tests/e2e/specs/multi-select.e2e.ts

```diff
@@ -135,7 +132,7 @@ const initMultiSelect = (page: Page, opt?: InitOptions): Promise<void> => {
   const selectOptions = [...'abc', ...(amount === 5 ? 'de' : '')]
     .map((x, idx) => {
       const attrs = [disabledIndex === idx ? 'disabled' : ''].join(' ');
-      const option = `<p-multi-select-option value="${x}" ${attrs}>Option ${x.toUpperCase()}</p-multi-select-option>`;
+      const option = `<p-multi-select-option ${x ? `value="${x}"` : ''} ${attrs}>Option ${x.toUpperCase()}</p-multi-select-option>`;
```

Comment:
How can `x` be falsy here?

Reply:
I thought about making the `selectOptions` configurable. With the current setup it can't be falsy, so I reverted the change